assets/js/settings.js

```diff
@@ -0,0 +1,18 @@
+document.getElementById( 'prpl-settings-form' ).addEventListener( 'submit', function( event ) {
+	event.preventDefault();
+	const form = new FormData( this );
+	const data = form.getAll( 'prpl-settings-post-types-exclude[]' );
+
+	// Save the options.
+	progressPlannerAjaxRequest( {
+		url: progressPlanner.ajaxUrl,
+		data: {
+			action: 'progress_planner_save_cpt_settings',
+			_ajax_nonce: progressPlanner.nonce,
+			include_post_types: data,
+		}
+	} );
+
+	document.getElementById( 'submit-exclude-post-types' ).disabled = true;
```

Comment:
We should change this to include to prevent future confusion :)

Reply:
You're right, I missed them in my previous commit. Fixed! 👍